api/PublicKeyCredentialCreationOptions.json

```diff
@@ -0,0 +1,484 @@
+{
+  "api": {
+    "PublicKeyCredentialCreationOptions": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PublicKeyCredentialCreationOptions",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": "67"
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "60"
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "rp": {
```

Comment:
The hierarchy here is messed up, all the subfeatures after `rp` are children of `rp`.

<img width="515" alt="screen shot 2018-06-09 at 3 55 49 pm" src="https://user-images.githubusercontent.com/2977353/41196449-a87a1cf4-6bfd-11e8-9fe4-9fe01654d9a8.png">

Reply:
Thanks for pointing this out, I have fixed the hierarchy now in 47f9b891b8708405b4a0bd26f6cb1c5a104ec927.